deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
The admin can optionally provide one, or there's a default. Either way they
have to label all the nodes for fluentd...

Before daemonsets the nodes were targeted by the DC being manually scaled
up to the number of nodes.

On Sun, Apr 24, 2016 at 8:58 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In deployment/deployer.yaml
> <https://github.com/openshift/origin-aggregated-logging/pull/119#discussion_r60855319>
> :
>
> > @@ -67,6 +67,16 @@ items:
> >        - watch
> >        - delete
> >        - update
> > +  - apiVersion: v1
>
> I would expect the admin to provide a label selector indicating what nodes
> they want fluentd to run on. How were the nodes being targeted before?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/119/files/6b633b5a527bad8144cb3bf6c21f5ee2b0575c61#r60855319>
>